### DISCUSSION TITLE
Show a Bunch is Active in the Menu Bar

### DISCUSSION THREAD

**Discussion by moffat** [ttscoff/bunch-docs#45] [Ideas]
As the subject says, it'd be nice to have a visual indication in the menu bar (no color please!) that a bunch (or bunches I guess) are active. Perhaps a hollow icon for no bunch active, filled for active? (not sure if this will work as the icon is quite thin--in a good way!). 

Many of my bunches turn off notifications so I can focus, and to be honest I often forget to "turn off" the bunch (still building up good habits about this). Having a visual cue would help.

**Comment by ttscoff** [comment]
You could always put `close after` timers on your Bunches, which you could set generously and which would act as a safety.

I always have at least 2 Bunches open at any given time, so I don't personally see much use in a top-level indicator that a Bunch is open. I know use cases vary significantly, but I think a timer is a more reliable option than a minor cosmetic change in the menu bar.

**Reply by moffat** [reply]
Good suggestion! I'll try that.
